Gruntfile.js

```diff
@@ -65,8 +64,14 @@ module.exports = function(grunt) {
           'modules/reliability/js/columnFormatter.js': 'modules/reliability/jsx/columnFormatter.js'
         }
       }
+    },
+
+    run: {
+      webpack: {
+        cmd: 'webpack'
```

Comment:
Are you using a global `grunt`? Not sure that would cause this, but try `./node_modules/.bin/grunt run:webpack`